src/OrchardCore.Modules/OrchardCore.Users/Permissions.cs

```diff
@@ -1,68 +1,61 @@
-using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 using OrchardCore.Security.Permissions;
-using OrchardCore.Security.Services;
 
 namespace OrchardCore.Users
 {
     public class Permissions : IPermissionProvider
     {
         public static readonly Permission ManageUsers = CommonPermissions.ManageUsers;
         public static readonly Permission ViewUsers = CommonPermissions.ViewUsers;
+        public static readonly Permission EditOwnUserInformation = new("ManageOwnUserInformation", "Edit own user information", new Permission[] { ManageUsers });
```

Comment:
EditOwnUserInformation != ManageOwnUserInformation

Reply:
This is just semantic. The key word "Manage" here really means "Edit". "Manage" could also include the ability to delete own info. This is why I would I would change the property name so it's more meaningful. but, I can change it.